manage_arkime/commands/create_cluster.py

```diff
@@ -22,12 +29,36 @@ def cmd_create_cluster(profile: str, region: str, name: str):
         constants.get_opensearch_domain_stack_name(name),
         constants.get_viewer_nodes_stack_name(name)
     ]
-    create_context = context.generate_create_cluster_context(name, cert_arn)
+    create_context = context.generate_create_cluster_context(name, cert_arn, capture_plan, ecs_resource_plan)
     cdk_client.deploy(stacks_to_deploy, aws_profile=profile, aws_region=region, context=create_context)
 
-def _set_up_viewer_cert(profile: str, region: str, name: str) -> str:
-    aws_provider = AwsClientProvider(aws_profile=profile, aws_region=region)
+def _get_capacity_plans(cluster_name: str, expected_traffic: int, aws_provider: AwsClientProvider) -> Tuple[CaptureNodesPlan, EcsSysResourcePlan]:
+    
+    if not expected_traffic:
+        try:
+            plan_json = ssm_ops.get_ssm_param_json_value(
+                constants.get_cluster_ssm_param_name(cluster_name),
+                "captureNodesPlan",
+                aws_provider
+            )
+            capture_plan = CaptureNodesPlan(
+                instance_type=plan_json["instanceType"],
```

Comment:
would it be possible to keep everything as camelCase, or not the way things are done?

Reply:
Don't have a strong reason other than trying to respect both language's conventions.  `instanceType` is a instance variable in a class on the TypeScript side, and we're serializing/deserializing the JSON into objects on both side.  Doing this minor translation enables us to keep the linters for both happy when we eventually turn them on.

That said, I'm probably going to be rejiggering how this capacity plan stuff is done for the OpenSearch Domain configuration task I'm working on next.  I'll take another look at this then.